README.md

```diff
@@ -33,7 +33,19 @@ described in detail below.
 
 Sign up for IBM's [Data Science Experience](http://datascience.ibm.com/). By signing up for the Data Science Experience, two services: ``DSX-Spark`` and ``DSX-ObjectStore`` will be created in your Bluemix account.
 
-## 4. Create the notebook
+## 2. Create the notebook
+
+Create the Project:
+* From the [IBM Data Science Experience page](https://apsportal.ibm.com/analytics) either click the "Projects" tab at the top or scroll down to "Recently updated projects".
+* Click on "+ Create Project" in the Projects tab view or "+ New Project" under Recently updated projects.
```

Comment:
must button-like things we highlight using back-ticks instead of quotes. That makes them a little buttony